en/flight_modes_fw/land.md

```diff
@@ -0,0 +1,49 @@
+# Land Mode (Fixed-wing)
+
+[<img src="../../assets/site/position_fixed.svg" title="Position estimate required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+The _Land_ flight mode causes the vehicle to descend at the position where the mode was engaged, following a circular path until touchdown.
+After landing, vehicles will disarm after a short timeout (by default).
+
+:::warning
+Fixed-wing _land mode_ should only be used in an **emergency**!
+The vehicle will descend around the current location irrespective of the suitability of the underlying terrain, and touch down while following a circular flight path.
+
+Where possible, instead use [Return mode](../flight_modes/return.md) with a predefined [Fixed-wing mission landing](../flight_modes/mission.md#fw-mission-landing).
+:::
+
+:::note
+
+- This mode requires a valid global position estimate (from GPS or inferred from a [local position](../ros/external_position_estimation.md#enabling-auto-modes-with-a-local-position)).
+- In a failsafe the mode only requires altitude (typically a barometer is built into the flight controller).
+- This mode is automatic - no user intervention is _required_ to control the vehicle.
+- RC control switches can be used to change flight modes on any vehicle.
+- RC stick movement is ignored.

```

Comment:
@sfuhrer Note, it is implied but not stated in the old version that FW land mode ignores stick input. Is that correct? I have added that it is here.